libraries/ui-library/src/utils/event-listeners.ts

```diff
@@ -55,9 +50,11 @@ export class EventListeners {
   }
 
   removeAll = () => {
-    while (this.eventListeners.length) {
-      const { el, name, listener } = this.eventListeners.pop();
-      el.removeEventListener(name, listener);
+    while (this.eventListeners.length > 0) {
+      const eventListener: EventListener | undefined = this.eventListeners.pop();
+      if (eventListener != null) {
+        eventListener.el.removeEventListener(eventListener.name, eventListener.listener);
+      }
```

Comment:
Could be simplified to 

```typescript
removeAll = () => {
    this.eventListeners
        .filter(eventListener => eventListener != null)
        .forEach(eventListener => eventListener.el.removeEventListener(eventListener.name, eventListener.listener));
};
```

Reply:
I thought about changing this too, but decided to keep it as is for this PR and postpone this and refactor this utility later.